src/Helpers.py

```diff
@@ -85,4 +87,37 @@ def _is_manualobject_enabled(multiworld: MultiWorld, player: int, object: any) -
             enabled = False
             break
 
-    return enabled
\ No newline at end of file
+    return enabled
+
+def get_items_for_player(multiworld: MultiWorld, player: int) -> List[Item]:
+    """Return list of items of a player including placed items"""
+    return [i for i in multiworld.get_items() if i.player == player]
+
+def get_items_with_value(world: World, multiworld: MultiWorld, value: str, player: Optional[int] = None, force: bool = False) -> dict[str, int]:
+    """Return a dict of every items with a specific value type present in their respective 'value' dict\n
+    Output in the format 'Item Name': 'value count'\n
+    Keep a cache of the result and wont redo unless 'force == True'
+    """
+    if player is None:
+        player = world.player
+
+    player_items = get_items_for_player(multiworld, player)
+    # Just a small check to prevent caching {} if items don't exist yet
+    if not player_items:
+        #raise Exception("get_items_with_value shouldn't be called before items are created.")
+        return {value: -1}

```

Comment:
idk if its better to raise Exception here or return empty dict when this is used too early in by a dev

Reply:
This only triggers when the user has no items defined, so I think it's okay to just return a <1 value. If they try to figure out why their item values aren't working, they should figure out that they have no items pretty quickly 😜